packages/next/build.mjs

```diff
@@ -18,11 +18,18 @@ await Promise.all([
   esbuild({
     entryPoints: [
       'src/legacy-launcher.ts',
-      'src/server-launcher.ts',
       'src/templated-launcher-shared.ts',
       'src/templated-launcher.ts',
     ],
   }),
+  // server-launcher imports some modules, so we need to bundle it
+  esbuild({
+    entryPoints: ['src/server-launcher.ts'],
+    bundle: true,
```

Comment:
Do we need to bundle this file or should we just add the new relative imports to the above list as well?

Reply:
IMHO bundling is the easiest option to reason about, we just get one file regardless of what it uses